### PR TITLE
Work around SR-13945 segfault.

### PR DESCRIPTION
**Pull request by pschuh**
See https://bugs.swift.org/browse/SR-13945 for more info.

### CODE REVIEW

**Comment by marcrasi** [comment]
looks like the test flaked while downloading something

**Review by marcrasi — APPROVED** [review]
Wow this looks difficult.

I hope someone fixes SR-13945 soon so that we can undo this.

Looks good to me as long as the tests pass.

**Comment by pschuh** [comment]
> Wow this looks difficult.
> 
> I hope someone fixes [SR-13945](https://bugs.swift.org/browse/SR-13945) soon so that we can undo this.
> 
> Looks good to me as long as the tests pass.

Should we consider disabling the test instead?

**Comment by marcrasi** [comment]
> Should we consider disabling the test instead?

Yeah, that sounds like it would be easier. Wordseg is going to be completely broken until we fix SR-13954, but I think no one is using wordseg these days anyways.

**Review by texasmichelle — APPROVED** [review]
I also tried a bunch of things to work around this and they were all ugly. Disabling until the upstream regression is fixed SGTM

File: Tests/TextTests/WordSegmentationTests/ProbeLayers.swift [hunk 1]
```diff
@@ -184,6 +184,8 @@ class WordSegProbeLayerTests: XCTestCase {
     let lattice = model.buildLattice(abab, maxLen: 5, device: device)
     XCTAssert(lattice.isAlmostEqual(to: Example1.lattice, tolerance: 1e-5))
 
+/*
+// TODO: renable once SR-13945 is fixed.
```
Comment (texasmichelle) [inline Tests/TextTests/WordSegmentationTests/ProbeLayers.swift:188]:
```suggestion
// TODO: re-enable once SR-13945 is fixed.
```

**Pull request merged**